bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/roomba/RoombaConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.irobot.roomba;
+
+/**
+ * Roomba Thing configuration
+ *
+ * @author Pavel Fedin <pavel_fedin@mail.ru> - Initial contribution
+ *
+ */
+public class RoombaConfiguration {
```

Comment:
Please add the NonNullByDefault annotation.

Reply:
The official doc says that these are not mandatory for data-only classes like config. But enjoy.